sdk/ml/azure-ai-ml/azure/ai/ml/_restclient/v2022_10_01_preview/models/_models_py3.py

```diff
@@ -19634,6 +19634,9 @@ class RegistryProperties(ResourceBase):
     :ivar region_details: Details of each region the registry is in.
     :vartype region_details:
      list[~azure.mgmt.machinelearningservices.models.RegistryRegionArmDetails]
+    :ivar managed_resource_group_tags: Tags to be applied to the managed resource group associated
+     with this registry.
+    :vartype managed_resource_group_tags: any
```

Comment:
Should this be `:vartype ...: dict[str, str]` ? Like what `tags` are

Reply:
Same comment in all other places that type the new field

Reply:
I'm not certain - this is the same type as other tags that I found already in the swagger.

Reply:
Just double checked - I was missing one additional property in my new swagger def compared to other tags. Adding that in changed the typing in the autorest to dict[str,str]. I just pushed a changed to include this.